docs/api-reference/next/future/image.md

```diff
@@ -39,13 +41,64 @@ Compared to `next/image`, the new `next/future/image` component has the followin
 - Removes `loader` config in favor of [`loader`](#loader) prop
 - Note: the [`onError`](#onerror) prop might behave differently
 
-The default layout for `next/image` was `intrinsic`, which would shrink the `width` if the image was larger than it's container. Since no styles are automatically applied to `next/future/image`, you'll need to add the following CSS to achieve the same behavior:
+## Migration
+
+Although `layout` is not available, you can migrate `next/image` to `next/future/image` using a few props. The following is a comparison of the two components:
 
-```css
-max-width: 100%;
-height: auto;
+**intrinsic: next/image**
```

Comment:
Should we use a table to organize the legacy/future comparison a bit more together similar to our page config allowed/not allowed table? https://github.com/vercel/next.js/blob/canary/errors/invalid-page-config.md

Reply:
Tables don't render nicely right now. For example https://nextjs.org/docs/api-reference/next/image#layout

Reply:
@ijjk Ok I think I got a decent table in [b69feb1](https://github.com/vercel/next.js/pull/39270/commits/b69feb136ec23f0c5efd9691c47a57a199411112)